packages/amplify_datastore/CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+## 0.3.0-rc.2 (2021-10-25)
+
+This release candidate requires to install `@aws-amplify/cli@6.2.0-custom-type-preview.0`
```

Comment:
I think the wording is a bit confusing here.  Could we say something like: This release candidate requires a preview version of amplify-cli that can be downloaded with this command: ""